README.md

```diff
@@ -25,28 +25,29 @@ Until now (2023/03/30) I have finished or barely finished these functionalities.
 - A Web Server
   + Show Keyboard Status
   + Change Keymap Dynamicly
+  + Support Dynamic Macros. Maros can be changed from the web interface.
   + OTA Update
 - I2C Display Support (LED controller 1306)
 
 ### Planned Features
 - Beautify Web GUI (use [numl.design](https://numl.design/))
-- Macro Definition Trough Web
 - Key Recording
-- Power Resume (esp32_keyboard can't wake up from USB suspend by now)
+- Support Wifi Configuration
 
 ### Build Process
-It requires [a modified esp-idf version 4.4.4](https://github.com/paul356/esp-idf) to build this project. The changes are adding tinyusb HID support, esp_lvgl_port and lvgl 8.3. Please note Epressif has removed tinyusb from esp-idf 5.0. So esp-idf 5.0 can't be used. After you have esp-idf [installed and configured](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#manual-installation), run following commands.
+It build with esp-idf 5.3. It used to build with a customized esp-idf 4.4 except for version v0.0.1. After you have esp-idf [installed and configured](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#manual-installation), run following commands. Note `get_idf` is the bash alias for enter esp-idf environment.
```

Comment:
It used to build with a customized esp-idf 4.4 for version v0.0.1. ...  is the bash alias for _entering_ esp-idf environment.